src/main/java/pageObjects/DevicePage.java

```diff
@@ -173,10 +278,23 @@ public void createDeviceWithWhiteSpacesNameField() {
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
     }
 
+    public void editDeviceWithWhiteSpacesNameField(){
+        successfullyCreateDevice();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_dropdown_" + deviceName)));
+        driver.findElement(By.id("device_dropdown_" + deviceName)).click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_edit_" + deviceName)));
+        driver.findElement(By.id("device_edit_" + deviceName)).click();
+        wait.until(ExpectedConditions.urlToBe(Credentials.device_details));
+        nameField.click();
+        nameField.clear();
+        nameField.sendKeys(DEVICE_NAME_WS);
+        saveDeviceButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
```

Comment:
why do we have a wait for the error message in this test but on the other negative edit tests we dont?